### PR TITLE
#366: model wrapper has a method to update all default values to the…

### PR DESCRIPTION
**Pull request by sideisra**
… current model values so that subsequent reset calls will reset all values to the current model values.

### CODE REVIEW

File: mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/mapping/ModelWrapper.java [hunk 1]
```diff
@@ -567,7 +590,17 @@ public void reset() {
 		
 		calculateDifferenceFlag();
 	}
-	
+
+	/**
+	 * Loads values from wrapped object again and sets the current value s default. Subsequent calls to {@link #reset()}
```
Comment (stefanil) [inline mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/mapping/ModelWrapper.java]: as default..

File: mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/mapping/ModelWrapper.java [hunk 2]
```diff
@@ -567,7 +590,17 @@ public void reset() {
...
+	 * will rest the values to this new default.
```
Comment (stefanil) [inline mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/mapping/ModelWrapper.java]: reset

**Comment by stefanil** [comment]
Except those grammar thangs in comments i'd like to approve this issue. But can't find where :)

**Comment by manuel-mauky** [comment]
Thanks for your pull request. I like the idea a lot.
However in my oppinion the name of the method is not intuitive enought.
I've change the name to `useCurrentValuesAsDefaults` so that the developer can immediately see what's going on. Additionally I've extended the Javadoc a little bit.

**Pull request merged**